### ISSUE TITLE
Google Api Error: invalid: Rollout fraction may only be used on production track.

### ISSUE THREAD

**Issue by ima747** [fastlane/fastlane#12372]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

When running supply to the alpha track it fails saying "Google Api Error: invalid: Rollout fraction may only be used on production track."

I'm assuming there was a google API change as this worked fine yesterday (albeit 1 version earlier on fastlane).

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[07:23:51]: --------------------
[07:23:51]: --- Step: supply ---
[07:23:51]: --------------------

+-------------------------+---------------------------------------------------------------------+
|                                   Summary for supply 2.93.0                                   |
+-------------------------+---------------------------------------------------------------------+
| track                   | alpha                                                               |
| skip_upload_apk         | false                                                               |
| apk                     | REDACTED/build/outputs/apk/release/REDACTED-release-play-store-signed.apk |
| mapping                 | fastlane/metadata/mapping/mapping_build_69.txt                      |
| skip_upload_metadata    | true                                                                |
| skip_upload_images      | true                                                                |
| skip_upload_screenshots | true                                                                |
| package_name            | com.REDACTED.android.REDACTED                                    |
| metadata_path           | ./fastlane/metadata/android                                         |
| json_key                | ~/.googleplay/Google Play Android Developer-9a6c7589888f.json       |
| validate_only           | false                                                               |
| check_superseded_tracks | false                                                               |
+-------------------------+---------------------------------------------------------------------+

[07:23:54]: Preparing to upload for language 'af'...
[07:23:55]: Preparing to upload for language 'ar'...
[07:23:55]: Preparing to upload for language 'bg'...
[07:23:55]: Preparing to upload for language 'ca'...
[07:23:55]: Preparing to upload for language 'cs-CZ'...
[07:23:55]: Preparing to upload for language 'da-DK'...
[07:23:55]: Preparing to upload for language 'de-DE'...
[07:23:55]: Preparing to upload for language 'el-GR'...
[07:23:56]: Preparing to upload for language 'en-AU'...
[07:23:56]: Preparing to upload for language 'en-CA'...
[07:23:56]: Preparing to upload for language 'en-GB'...
[07:23:56]: Preparing to upload for language 'en-US'...
[07:23:56]: Preparing to upload for language 'en-ZA'...
[07:23:56]: Preparing to upload for language 'es-419'...
[07:23:56]: Preparing to upload for language 'es-ES'...
[07:23:57]: Preparing to upload for language 'es-US'...
[07:23:57]: Preparing to upload for language 'et'...
[07:23:57]: Preparing to upload for language 'fa'...
[07:23:57]: Preparing to upload for language 'fi-FI'...
[07:23:57]: Preparing to upload for language 'fr-CA'...
[07:23:57]: Preparing to upload for language 'fr-FR'...
[07:23:57]: Preparing to upload for language 'hi-IN'...
[07:23:57]: Preparing to upload for language 'hr'...
[07:23:58]: Preparing to upload for language 'hu-HU'...
[07:23:58]: Preparing to upload for language 'id'...
[07:23:58]: Preparing to upload for language 'it-IT'...
[07:23:58]: Preparing to upload for language 'iw-IL'...
[07:23:58]: Preparing to upload for language 'ja-JP'...
[07:23:58]: Preparing to upload for language 'ko-KR'...
[07:23:58]: Preparing to upload for language 'lt'...
[07:23:59]: Preparing to upload for language 'lv'...
[07:23:59]: Preparing to upload for language 'ms'...
[07:23:59]: Preparing to upload for language 'nl-NL'...
[07:23:59]: Preparing to upload for language 'no-NO'...
[07:23:59]: Preparing to upload for language 'pl-PL'...
[07:23:59]: Preparing to upload for language 'pt-BR'...
[07:23:59]: Preparing to upload for language 'pt-PT'...
[07:23:59]: Preparing to upload for language 'ro'...
[07:24:00]: Preparing to upload for language 'ru-RU'...
[07:24:00]: Preparing to upload for language 'sk'...
[07:24:00]: Preparing to upload for language 'sl'...
[07:24:00]: Preparing to upload for language 'sr'...
[07:24:00]: Preparing to upload for language 'sv-SE'...
[07:24:00]: Preparing to upload for language 'sw'...
[07:24:00]: Preparing to upload for language 'th'...
[07:24:01]: Preparing to upload for language 'tr-TR'...
[07:24:01]: Preparing to upload for language 'uk'...
[07:24:01]: Preparing to upload for language 'vi'...
[07:24:01]: Preparing to upload for language 'zh-CN'...
[07:24:01]: Preparing to upload for language 'zh-HK'...
[07:24:01]: Preparing to upload for language 'zh-TW'...
[07:24:01]: Preparing apk at path 'REDACTED/build/outputs/apk/release/REDACTED-release-play-store-signed.apk' for upload...
[07:24:12]: Updating track 'alpha'...
[07:24:12]: Uploading all changes to Google Play...
+-----------------------------+---------------------------------------------------------------------+
|                                           Lane Context                                            |
+-----------------------------+---------------------------------------------------------------------+
| DEFAULT_PLATFORM            | android                                                             |
| PLATFORM_NAME               | android                                                             |
| LANE_NAME                   | android beta                                                        |
| GIT_REPO_WAS_CLEAN_ON_START | true                                                                |
| SIGNED_APK_PATH             | REDACTED/build/outputs/apk/release/REDACTED-release-play-store-signed.apk |
+-----------------------------+---------------------------------------------------------------------+
[07:24:14]: Google Api Error: invalid: Rollout fraction may only be used on production track.</pre>
</details>

**Comment by Nick-Lucas** [comment]
+1 we're blocked by this too

**Comment by barrynorman** [comment]
Same here!

**Comment by joshdholtz** [comment]
Heyyyy 👋 Looking into right now!

**Comment by mindofai** [comment]
Same issue.

**Comment by joshdholtz** [comment]
Got the solution! Doing a bit more testing then we will get a new version released for your all 👍

**Comment by maxired** [comment]
awesome @joshdholtz .
maybe we can also patch locally to test it ?

**Comment by joshdholtz** [comment]
@maxired For sure! If you could change your `Gemfile` to...
```rb
gem 'fastlane', :git => 'https://github.com/fastlane/fastlane.git', :branch => 'joshdholtz-fix-supply-rollout'
```

and run `bundle install` and let me know if things worked that would be 💯  😊

**Comment by joshdholtz** [comment]
I'm not sure if that thumbs up means "You understand" or "it worked" lol 🙃

**Comment by maxkomarychev** [comment]
@joshdholtz any way to install gem from specific branch for those who aren't using gemfile?

**Comment by joshdholtz** [comment]
@maxkomarychev Not easily 😔 Requires pulling down the source, building a gem from it, and installing it 😬

**Comment by maxired** [comment]
@joshdholtz sorry, meant currently testing ;) seems to work, but let me retry with a full build

**Comment by markwk** [comment]
Fix worked for me on Bitrise build with Fastlane. Great and fast work @joshdholtz Much thanks!

**Comment by joshdholtz** [comment]
Until `2.93.1` is able to be released, you can all change your `Gemfile` to...
```rb
gem 'fastlane', :git => 'https://github.com/fastlane/fastlane.git'
```

**Comment by Nick-Lucas** [comment]
Hey @joshdholtz when do you expect `2.93.1` to be released? Our build system is tricky to change this, so we want to figure out priorities.

Thanks a lot for your fast work on the issue!

**Comment by joshdholtz** [comment]
@Nick-Lucas The process is happening right now. Should be within 10 minutes 😁

**Comment by ks32** [comment]
@ima747 same error for Internal track on Google Play Store

**Comment by joshdholtz** [comment]
Alright alright alright! `2.93.1` is released now with the fix for this 👞 🐛

**Comment by estebansolis** [comment]
same error for internal track :(

**Comment by joshdholtz** [comment]
@estebansolis Upgrading to `2.93.1` (released about 2 hours ago) should solve this for you

**Comment by estebansolis** [comment]
You @joshdholtz  are a genius.

**Comment by viniciusoh** [comment]
It also solved the erroneous error message "[!] Google Api Error: rolloutNotCompleted: Cannot modify the production track until you complete the rollout." on 2.80.0 (fastlane-2.80.0)

Thank you! 😄

**Comment by qaisammari** [comment]
faced the same issue, applied the fix and it is working fine now , thanks

**Comment by deepeeess** [comment]
Yeah this fix works 💯

**Comment by singletondejan** [comment]
2.93.1 fixed the issue. Thanks!
